### DISCUSSION TITLE
Is it possible to send QUIC packets?

### DISCUSSION THREAD

**Discussion by lya1016** [dannagle/PacketSender#377] [Q&A] [answered]
Can I select which protocol to use? Such as QUIC？

**Comment by dannagle** [comment, marked as answer]
Your options are in the drop-down. They are UDP, TCP, SSL, and various HTTP methods. 

QUIC is built on top of UDP, so you could in theory communicate with a QUIC server, but you would only get as far as the initial handshake sequence.

Feel free to open a feature request. The are efforts to include more built-in protocols.
